src/uct/api/v2/uct_v2.h

```diff
@@ -963,6 +1023,21 @@ ucs_status_t uct_ep_connect_to_ep_v2(uct_ep_h ep,
                                      const uct_ep_addr_t *ep_addr,
                                      const uct_ep_connect_to_ep_params_t *params);
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Checks if an endpoint is connected to a remote address.
+ *
+ * This function checks if a local endpoint is connected to a remote address.
+ *
+ * @param [in] ep      Endpoint to check.
+ * @param [in] params  Parameters as defined in @ref
+ *                     uct_ep_is_connected_params_t.
+ *
+ * @return Nonzero if connected, 0 otherwise.
+ */
+int uct_ep_is_connected(uct_ep_h ep,
```

Comment:
why int and not ucs_status_t

Reply:
because it returns a boolean value (connected or not connected). similar to is_reachable